swupd/rename.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package swupd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+)
+
+// BSDIFFSIZE is smallest size of file to consider using bsdiff
+const BSDIFFSIZE = 200
```

Comment:
Optional: I've added a similar constant (but with a more detailed name and background story) in the delta integration PR. If that PR lands first, reuse it.